kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ResourcesUtil.java

```diff
@@ -224,4 +226,18 @@ static <O extends HasMetadata, R extends HasMetadata> Set<ResourceID> findReferr
                 owner,
                 primary -> ResourcesUtil.isReferent(refAccessor.apply(primary), referent));
     }
+
+    /**
+     * Generates the minimal set of metadata require for ServerSideApply that is the co-ordinates of the resource.
+     * @param resource the resource we want to copy the Co-ordinates from
+     * @return An ObjectMeta with the co-ordinates specified (and nothing else)
+     * @param <R> the type source resource (with existing Metadata) to copy the co-ordinates from.
+     */
+    public static <R extends HasMetadata> ObjectMeta coordinatesMetadata(R resource) {
```

Comment:
Could equally well be call `minimalMetadata` but I thought this was a bit more indicative of the intended usage.